mojave/code/modules/projectiles/boxes_magazines/ammo_stacking.dm

```diff
@@ -85,34 +85,6 @@
 		add_overlay(bullet)
 	return UPDATE_ICON_STATE | UPDATE_OVERLAYS
 
-/obj/item/ammo_box/magazine/ammo_stack/attackby(obj/item/A, mob/user, params, silent = FALSE, replace_spent = 0)
-	var/num_loaded = 0
-	if(!can_load(user))
-		return
-	if(istype(A, /obj/item/ammo_box))
-		var/obj/item/ammo_box/AM = A
-		for(var/obj/item/ammo_casing/AC in AM.stored_ammo)
-			var/did_load = give_round(AC, replace_spent)
-			if(did_load)
-				AM.stored_ammo -= AC
-				num_loaded++
-			if(!did_load || !multiload)
-				break
-		if(num_loaded)
-			AM.update_ammo_count()
-	if(istype(A, /obj/item/ammo_casing))
-		var/obj/item/ammo_casing/AC = A
-		if(give_round(AC, replace_spent))
-			user.transferItemToLoc(AC, src, TRUE)
-			num_loaded++
-			AC.update_appearance()
-
-	if(num_loaded)
-		if(!silent)
-			to_chat(user, span_notice("You load [num_loaded] shell\s into \the [src]!"))
-			playsound(src, 'sound/weapons/gun/general/mag_bullet_insert.ogg', 60, TRUE)
-		update_ammo_count()
-
```

Comment:
i dont remember why i did this without calling parent but i am 100% sure it was for an important reason

Reply:
Already in commit upper

Reply:
and that same code, bruh